api/app_analytics/views.py

```diff
@@ -32,6 +40,59 @@
 logger = logging.getLogger(__name__)
 
 
+class SDKAnalyticsFlagsV2(CreateAPIView):
+    permission_classes = (EnvironmentKeyPermissions,)
+    authentication_classes = (EnvironmentKeyAuthentication,)
+    serializer_class = SDKAnalyticsFlagsSerializer
+
+    def create(self, request: Request, *args, **kwargs) -> Response:
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        self.evaluations = serializer.validated_data["evaluations"]
+        if not self._is_data_valid():
+            # Mirror v1 implementation and return 200 to avoid
+            # breaking client integrations.
+            return Response(
+                {"detail": "Invalid data. Not logged."},
+                content_type="application/json",
+                status=status.HTTP_200_OK,
+            )
+        if settings.USE_POSTGRES_FOR_ANALYTICS:
+            track_feature_evaluation_v2.delay(
+                args=(
+                    request.environment.id,
+                    self.evaluations,
+                )
+            )
+        elif settings.INFLUXDB_TOKEN:
+            track_feature_evaluation_influxdb_v2(
+                request.environment.id, self.evaluations
+            )
```

Comment:
This should be a task as well.

Reply:
I mirrored the existing function but since this one should be a task then both of them should be, right? So I've updated both versions to operate as tasks. Let me know if this is a problem.